include/convex_bodies/orderpolytope.h

```diff
@@ -0,0 +1,741 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2012-2020 Vissarion Fisikopoulos
+// Copyright (c) 2018-2020 Apostolos Chalkis
+// Copyright (c) 2020-2021 Vaibhav Thakkar
+
+// Contributed and/or modified by Vaibhav Thakkar, as part of Google Summer of Code 2021 program.
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+#ifndef ORDER_POLYTOPE_H
+#define ORDER_POLYTOPE_H
+
+#include <iostream>
+#include "poset.h"
+#include <Eigen/Eigen>
+#include "preprocess/max_inscribed_ball.hpp"
+#ifndef VOLESTIPY
+    #include "lp_oracles/solve_lp.h"
+#endif
+
+
+template <typename Point>
+class OrderPolytope {
+public:
+    typedef Point PointType;
+    typedef typename Point::FT NT;
+    typedef Eigen::Matrix<NT, Eigen::Dynamic, 1> VT;
+    typedef Eigen::Matrix<NT, Eigen::Dynamic, Eigen::Dynamic> MT;
+    
+private:
+    Poset poset;
```

Comment:
I used "_" before those variables which are not core in representing the object, like for a poset, both `n` and `order_relations` are core for representing a poset. Similarly, in `order_polytope` only `b` is core for representation as `A` is not required in a matrix form, that's why `_A` is used. I used this because in `hpolytope.h` file, `_d` is used for representing the dimension variable.

Reply:
There is a lack of specific coding style in volesti and this is a problem we cannot solve here ;) So maybe it doesn't make sense to try to impose some style in the review. 
My opinion is that it is a bit unclear to declare variables by whether or not they are "core". 
FYI google style proposes a trailing underscore in all member variables https://google.github.io/styleguide/cppguide.html#Variable_Names 
Maybe it is a good time to start a discussion on a coding style. 
We can discuss this in an separate issue